db/src/bootstrap.rs

```diff
@@ -138,6 +152,16 @@ lazy_static! {
             .ok_or(ErrorKind::BadBootstrapDefinition("Unable to parse V2_SYMBOLIC_SCHEMA".into()))
             .unwrap()
     };
+
+    static ref V3_SYMBOLIC_SCHEMA: Value = {
+        let s = r#"{}"#;
```

Comment:
Correct, and you're welcome to just clone the `V2_SYMBOLIC_SCHEMA` or whatever is needed if you prefer.

The symbolic schema would/will change if/when we make `:db/txInstant` have `:db.valueType :db.type/inst`, like Datomic.